verticles/graphql/src/main/java/com/gentics/mesh/graphql/dataloader/NodeDataLoader.java

```diff
@@ -352,12 +373,17 @@ public boolean equals(Object o) {
 			if (this == o) return true;
 			if (!(o instanceof Context)) return false;
 			Context context = (Context) o;
-			return type == context.type && Objects.equals(languageTags, context.languageTags);
+			return type == context.type 
+					&& Objects.equals(languageTags, context.languageTags) 
+					&& Objects.equals(paging.toString(), context.paging.toString()) 
```

Comment:
Paging and native filter have to be compared by value, not by instance.

Reply:
paging.toString() will return the paging parameters as query string, but as far as I have seen, there is no guarantee about the order of the parameters in the resulting string